component-tools/src/test/java/org/talend/sdk/component/tools/DitaDocumentationGeneratorTest.java

```diff
@@ -51,22 +57,57 @@ void generateDita(@TempDir final File temporaryFolder, final TestInfo info) thro
         final Map<String, String> files = new HashMap<>();
         try (final ZipInputStream zip = new ZipInputStream(new FileInputStream(output))) {
             ZipEntry nextEntry;
+            int x = 1;
             while ((nextEntry = zip.getNextEntry()) != null) {
-                files.put(nextEntry.getName(), IO.slurp(zip));
+                String content = IO.slurp(zip);
+                String name = nextEntry.getName();
+                files.put(name, content);
+                BufferedWriter writer =
+                        new BufferedWriter(new FileWriter("/tmp/dita/" + new File(name).getName() + ".xml"));
+                x++;
+                writer.write(content);
+                writer.flush();
+                writer.close();
             }
         }
-        try (final BufferedReader reader = resource("generateDita1.xml")) {
-            assertEquals(reader.lines().collect(joining("\n")), files.get("generateDita/test/my.dita").trim());
-        }
-        try (final BufferedReader reader = resource("generateDita2.xml")) {
-            assertEquals(reader.lines().collect(joining("\n")), files.get("generateDita/test/my2.dita").trim());
-        }
+
+        assertTrue(false); // Revoir complètement ce test !
+
         assertEquals(4, files.size());
         // folders
         assertEquals("", files.get("generateDita/test/"));
         assertEquals("", files.get("generateDita/"));
     }
 
+    @ParameterizedTest
+    @CsvSource(value = { "nolayout,NoLayout.dita", "allbasic,AllBasic.dita", "alladvanced,AllAdvanced.dita",
```

Comment:
No test without datatstore or without dataset etc?

Reply:
No, can we have a connector without dataset/datastore ?